### DISCUSSION TITLE
The Play CDN values seem to override my custom overrides

### DISCUSSION THREAD

**Discussion by avanderhoek** [tailwindlabs/tailwindcss#7128] [Help] [answered]
Hi there,

I have a question on how to use the Play CDN in development mode together with my custom values.

Here are my stylesheets:
```
    {% block head_stylesheets %}
        {# Only use this for development #}
        {% if app.environment == 'dev' %}
            <script src="{{ asset('assets/cdn_tailwindcss_3.0.12') }}"></script>
        {% endif %}

        <link rel="stylesheet" href="{{ asset('assets/main.css') }}">
    {% endblock %}
```

And I have this in my Tailwind config file:
```
module.exports = {
    theme: {
        fontSize: {
            'xs': '.75rem !important', // 12px, same as TailwindCSS default
            'sm': '.875rem !important', // 14px, same as TailwindCSS default
            'base': '1rem !important', // 16px, same as TailwindCSS default
            'lg': '1.125rem !important', // 18px, same as TailwindCSS default
            'xl': '1.25rem !important', // 20px, same as TailwindCSS default
            '2xl': '1.5rem !important', // 24px, same as TailwindCSS default
            '3xl': '1.875rem !important', // 30px
            '4xl': '2.25rem !important', // 36px, same as TailwindCSS default
            '4.5xl': '2.5rem !important', // 40px, class introduced by us
            '5xl': '3rem !important', // 48px, same as TailwindCSS default
            '5.5xl': '3.5rem !important', // 56px, class introduced by us
            '6xl': '4rem !important', // 64px, sorry had to use the !important to override the Play CDN js class until we find a better solution
            '7xl': '5rem !important', // 80px, sorry had to use the !important to override the Play CDN js class until we find a better solution
            '8xl': '6.25rem !important', // 100px, sorry had to use the !important to override the Play CDN js class until we find a better solution
            '9xl': '7.5rem !important', // 120px, sorry had to use the !important to override the Play CDN js class until we find a better solution
        },
```
Right now it's the only way I know to use both my custom values and the CDN together. Otherwise I still get the CDN values with the default Tailwind values. I dont't want to use `!important` unless I only have to. Now I had to because the responsive classes got into trouble. This is probably not how you intended it.

Is it clear what I am asking? Does anyone have some advice on how to avoid having to use these kind of overrides in the config?

**Comment by adamwathan** [comment, marked as answer]
Hey! The Play CDN doesn't read your config file, you have to provide your config as a script directly on your page for it to work:

![image](https://user-images.githubusercontent.com/4323180/150152943-7e8894d3-f082-43aa-b953-690a2aae3e8d.png)

https://tailwindcss.com/docs/installation/play-cdn

Does that help?

**Reply by avanderhoek** [reply]
Thank you for your quick response! We are probably not in the same timezone, it is 4:50PM here 😄 . Your answer did help! It helped us understand that we need to figure out a way to get the tailwind config available to work with the Play CDN in dev mode and to get it in our CSS build without having to write the config twice. So we know how to move forward now. Again, thank you very much 🙏
